app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class SessionsController < ApplicationController
+  def new
+    redirect_to missions_path if session[:current_user_id].present?
+  end
+
+  def create
+    user = User.find_by(email: params[:email])
+
+    if user&.authenticate(params[:password])
+      session[:current_user_id] = user.id
+      redirect_to missions_path, notice: t('.success', name: user.name)
+    else
+      redirect_to login_path, notice: t('.failure')
+    end
+  end
+
+  def destroy
+    if session[:current_user_id]
+      session[:current_user_id] = nil
+      redirect_to login_path, notice: t('.success')
+    else
+      redirect_to missions_path, notice: t('.failure')
+    end
```

Comment:
使用 guard clause 應該可以讓邏輯（意圖）更明顯
```suggestion
    return redirect_to missions_path, notice: t('.failure') if session[:current_user_id].blank?

    session[:current_user_id] = nil
    redirect_to login_path, notice: t('.success')
```